cmw-dto/src/main/java/com/liberologico/cloudesire/cmw/model/dto/EnvironmentDTO.java

```diff
@@ -20,343 +23,262 @@
 import java.util.Objects;
 import java.util.Set;
 
+@ApiModel( description = "Platform configuration" )
 public class EnvironmentDTO extends DTO
 {
-    /**
-     * The name of this environment
-     */
+    @ApiModelProperty( "The name of this environment" )
```

Comment:
li ho cambiati tutti in una botta sola con trova-sostituisci di intellij con le regex lol